src/task.test.js

```diff
@@ -1,5 +1,6 @@
 import 'jest-localstorage-mock';
-import { addTask } from './task.js';
+import { addTask, updateTaskValue } from './task.js';
+import AddCheckBox from './completion.js';
 
```

Comment:
- [OPTIONAL] A good project directory structure convention is to wrap up all tests files inside `__tests__` directory 👍🏼